x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilities_finding_flyout/vulnerability_finding_flyout.tsx

```diff
@@ -71,6 +80,7 @@ export const VulnerabilityFindingFlyout = ({
   const renderTabs = () =>
     tabs.map((tab, index) => (
       <EuiTab
+        data-test-subj={`vulnerability-finding-flyout-tab-${tab.id}`}
```

Comment:
preferable to import from the `test_subjects` file

Reply:
Nice catch!